### PR TITLE
Update debezium/example-mysql docker image and ignore some tests for arm64 [HZ-1641]

### PR DESCRIPTION
**Pull request by Patras3**
Update `debezium/example-mysql:1.9.5.Final` to `debezium/example-mysql:1.9.6.Final`
Ignore tests with `debezium/example-postgres:1.7` - there is no working image of `example-postgres` for arm64

Checklist:
- [X] Labels (`Team:`, `Type:`, `Source:`, `Module:`) and Milestone set
- [X] Label `Add to Release Notes` or `Not Release Notes content` set
- [X] Request reviewers if possible

### CODE REVIEW

**Comment by Patras3** [comment]
run-with-arm64

**Comment by Patras3** [comment]
run-with-arm64

**Comment by Patras3** [comment]
run-with-arm64

File: extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/AbstractMySqlCdcIntegrationTest.java [hunk 1]
```diff
@@ -50,8 +50,10 @@ public abstract class AbstractMySqlCdcIntegrationTest extends AbstractCdcIntegra
                     .withPassword("mysqlpw")
     );
 
-    @Before
-    public void ignoreOnJdk15OrHigher() {
+    @BeforeClass
+    public static void ignoreOnArm64OrOnJdk15OrHigher() {
+        //There is no arm64 version of currently used docker image
+        assumeNoArm64Architecture();
```
Comment (TomaszGaweda) [inline extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/AbstractMySqlCdcIntegrationTest.java:56]: IMHO It's better to upgrade image to 1.9.6, which has ARM images

Comment (Patras3) [reply extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/AbstractMySqlCdcIntegrationTest.java:56]: I have created a task to update debezium separately: https://github.com/hazelcast/hazelcast/issues/23491
Currently, we are using `<debezium.version>1.9.5.Final</debezium.version>` versions should be aligned I think.

Comment (TomaszGaweda) [reply extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/AbstractMySqlCdcIntegrationTest.java:56]: that should not matter, should work just fine when upgrading to minor version

**Comment by Patras3** [comment]
run-lab-run

**Comment by Patras3** [comment]
run-with-arm64

**Comment by Patras3** [comment]
run-with-arm64

**Comment by Patras3** [comment]
run-arm64

File: extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/AbstractMySqlCdcIntegrationTest.java (hunk 1 above)
Comment (Patras3) [reply extensions/cdc-mysql/src/test/java/com/hazelcast/jet/cdc/mysql/AbstractMySqlCdcIntegrationTest.java]: I updated image of mysql, but for postgres there is no working version (even if in theory supports arm64).

File: extensions/kinesis/src/test/java/com/hazelcast/jet/kinesis/KinesisIntegrationTest.java [hunk 2]
```diff
@@ -87,6 +87,8 @@ public KinesisIntegrationTest() {
     @BeforeClass
     public static void beforeClass() {
         assumeDockerEnabled();
+        //Newer version of localstack with arm64 support fails in KinesisIntegrationTest
+        assumeNoArm64Architecture();
```
Comment (frant-hartm) [inline extensions/kinesis/src/test/java/com/hazelcast/jet/kinesis/KinesisIntegrationTest.java:91]: Do you have some issue or details what fails exactly?

File: extensions/cdc-postgres/src/test/java/com/hazelcast/jet/cdc/postgres/AbstractPostgresCdcIntegrationTest.java [hunk 3]
```diff
@@ -60,6 +61,12 @@ public abstract class AbstractPostgresCdcIntegrationTest extends AbstractCdcInte
                     .withStartupTimeoutSeconds(300)
     );
 
+    @BeforeClass
+    public static void ignoreOnArm64() {
+        //There is no working arm64 version of example-postgres image
```
Comment (frant-hartm) [inline extensions/cdc-postgres/src/test/java/com/hazelcast/jet/cdc/postgres/AbstractPostgresCdcIntegrationTest.java:66]: Can you please create
 an issue to start using official postgres images in the CDC tests.

**Review by frant-hartm — APPROVED** [review]

**Review by kwart — APPROVED** [review]

File: extensions/kinesis/src/test/java/com/hazelcast/jet/kinesis/KinesisIntegrationTest.java (hunk 2 above)
Comment (Patras3) [reply extensions/kinesis/src/test/java/com/hazelcast/jet/kinesis/KinesisIntegrationTest.java:91]: Created https://github.com/hazelcast/hazelcast/issues/23692

File: extensions/cdc-postgres/src/test/java/com/hazelcast/jet/cdc/postgres/AbstractPostgresCdcIntegrationTest.java (hunk 3 above)
Comment (Patras3) [reply extensions/cdc-postgres/src/test/java/com/hazelcast/jet/cdc/postgres/AbstractPostgresCdcIntegrationTest.java:66]: Created in Jira:
https://hazelcast.atlassian.net/browse/HZ-2060
I can adjust that in the future.

**Pull request merged**